### DISCUSSION TITLE
utf-8 cant decode byte 0x96 error in kohya_ss

### DISCUSSION THREAD

**Discussion by MightyNo22** [bmaltais/kohya_ss#2161] [General]
i keep getting this utf-8 cant decode byte 0x96 error while training with kohya_ss

i reinstall python, reinstall my pc, changed training images, training names. still cant fix this. please help me. 

![image](https://github.com/bmaltais/kohya_ss/assets/165064699/392febeb-9768-4f05-82ed-f49929255521)

**Comment by 5KilosOfCheese** [comment]
This is probably relevant https://github.com/bmaltais/kohya_ss/discussions/1744 
Make sure your folder paths, text captions, filenames, and sample prompts, and everything uses only letters and symbols present in Latin1 set.

For some reason the system can not read some kind of text that you give it.
